task09-react/src/components/ObjectSection.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import MysticObject from './MysticObject';
+import object from '../assets/img/object.svg'
+
+class ObjectSection extends Component {
+  render() { 
+    const objectName = "Gomu Gomu No Mi";
+    const description = "Is a Paramecia-type Devil Fruit that gives the user's body the properties of rubber, making the user a Rubber Human."
+                      + "It was then accidentally eaten by the series protagonist, Monkey D. Luffy."
+    return (
+      <article className="article article--mist">
+      <div className="article__container article__container--mist">
+        <h3 className="article__title article__title--mist">Mist Objects</h3>
+        <a href="#" className="article__link article__link--mist">See all</a>
+      </div>
+      <div className="article__flex article__flex--mist">
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
+        <MysticObject name="Gomu Gomu No Mi" img={object} alt="Gomu Gomu No Mi">
+          {description}
+        </MysticObject>
```

Comment:
Let's try to put this into a loop so there aren't repeated lines in here, also replace the alt value with the objectName variable in the img tags,

Reply:
True, I forgot to use the variable I declared lines above. Also repeating those lines doesn't look good and makes it tedious to add or remove objects as they are hard coded.